kallisto/Makefile

```diff
@@ -2,14 +2,14 @@
 build_output = runtime/kallisto
 runtime_fullpath = $(realpath runtime)
 build_tool = runtime-container.DONE
-git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../kallisto | cut -f1 -d " ")
+git_commit = a231803
```

Comment:
This is a bit confusing, but revert to the previous line. The git commit refers to _this repos commit hash_ not the commit of the tool you're pulling from.